packages/plugin-ext/src/common/plugin-api-rpc.ts

```diff
@@ -2173,6 +2173,9 @@ export interface TestingExt {
     /** Configures a test run config. */
     $onConfigureRunProfile(controllerId: string, profileId: string): void;
 
+    /** Sets the default on a given run profile */
+    $onChangeDefault(controllerId: string, profileId: string, isDefault: boolean): void;
```

Comment:
Not really correct according to our naming conventions ("onDidChange..."), but 1. it's consistent with the rest of the interface and 2. I invented it 🤦 .